tasty/skyspark/client.py

```diff
@@ -0,0 +1,60 @@
+import requests
+
+# TODO:
+# make parameters instance variables
+# construct query method
+
+
+class SkysparkClient:
+
+    # -- Class Constructor --
+    def __init__(self, api_url_endpoint):
+        self.api_url_endpoint = api_url_endpoint
+
+    def make_get_request(self, query_string: str, format: str = 'json'):
+        """
+        Make a get request to a (Skyspark) api endpoint. Note that the query string must be a valid
+        'axon' query.
+
+        :param query_string: the (axon) query string to include in the request
+        :type query_string: str
+        :param format: the format in which the response should be returned options are
+            ['json' , 'turtle' , 'csv' , 'zinc' , 'trio' , 'json-ld'] - the default is json
+        :type format: str
+        :return: the get response
+        :rtype: requests.Response Object
+        """
+
+        # set "Accept" header value based on specified format
```

Comment:
This works for sure but generally, when switch cases get longer than 3+ it begs for a key-value pair structure that maps the two values.  This would look something like this:
```
mapping_dict= {
    "turtle": "text/turtle",
    "csv": "text/csv",
    "json": "application/json"
}
```
That way you can just use the format variable as a key lookup into the dict and get its value pair! Something like `accept_type = mapping_dict[format]`

Reply:
Cool, yea that makes sense. How should I handle the default case / when the "format" input does not match a key in the dict?

Reply:
Is this a PEP8 recommendation. I didn't see it with a quick skim. A dict is definitely more concise though.